pkg/cli/cli.go

```diff
@@ -97,19 +96,27 @@ var (
     To see the full list of commands supported, run 'oc --help'.`)
 )
 
-func NewDefaultOcCommand(in io.Reader, out, errout io.Writer) *cobra.Command {
-	cmd := NewOcCommand(in, out, errout)
+var defaultConfigFlags = genericclioptions.NewConfigFlags(true).WithDiscoveryBurst(350).WithDiscoveryQPS(50.0)
```

Comment:
see more about why we shouldn't have this https://github.com/kubernetes/kubernetes/pull/117668#discussion_r1194742874

Reply:
thanks, I have moved this to a function - also added `WithDeprecatedPasswordFlag` which is missing as compared to the kubectl